### ISSUE TITLE
Web Deployment [FEATURE REQUEST]

### ISSUE THREAD

**Issue by damare01** [Flameish/Novel-Grabber#344]
**Additional context**

Any possibility that this project can be or will be able to deploy as a web?

**Comment by Flameish** [comment]
Yes, but I've only recently started to work on this project again after a hiatus, so no guarantees to a release date as I still don't have much time to work on NG.
